### PR TITLE
Fix package contents

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

**Review by heapwolf — APPROVED** [review]
lgtm

File: package.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
   "type": "module",
   "scripts": {
     "test": "standard index.js",
-    "publish": "npm publish && npm publish --registry https://npm.pkg.github.com",
+    "pub": "npm pub && npm publish --registry https://npm.pkg.github.com",
```
Comment (jwerle) [inline package.json:12]: Should be `npm run pub` right?

Comment (jwerle) [reply package.json:12]: Or wait what does `npm pub` do

Comment (chicoxyzzy) [reply package.json:12]: it's an alias for `npm publish`. I just copied this line from the `io`'s package.json

**Pull request merged**